tensorboard/functionaltests/wct_test_driver.py

```diff
@@ -90,13 +117,14 @@ def tearDown(cls):
       cls.process.wait()
 
     def test(self):
-      driver = webtest.new_webdriver_session()
+      driver = webtest.new_webdriver_session(
+          capabilities={'loggingPrefs':{'browser':'ALL'}}
+      )
       url = "http://localhost:%s%s" % (self.port, web_path)
       driver.get(url)
-      wait.WebDriverWait(driver, 10).until(
-          expected_conditions.title_contains("test"))
-      title = driver.title
-      if "failing test" in title or "passing test" not in title:
-        self.fail(title)
+      browser_log_indicates_result = BrowserLogIndicatesResult()
+      wait.WebDriverWait(driver, 10).until(browser_log_indicates_result)
+      if not browser_log_indicates_result.passed:
```

Comment:
`self.assertTrue(browser_log_indicates_result.passed)` is simpler, yes?

Reply:
I had that initially, but then the error message for a passing test is something like 'False' is 'True', which I didn't like so much. It's only slightly better with 'fail'.

Reply:
Got it; thanks.